COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
+directive.
+{:.note}
+
+### Add a dependency between a Kotlin Pod and Xcode project with one target
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`) 
+of your Kotlin project.  
+    This step helps synchronize your Xcode project with Kotlin Pod dependencies by calling `pod install` for your `Podfile`.
+3. Specify the minimum target version for the Pod library.
+    > If you don't specify the minimum target version and a dependency Pod requires a higher deployment target, you may get an error.
+    {:.note}
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+        
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            ios.deploymentTarget = "13.5"
+            pod("AFNetworking", "~> 4.0.0")
+            podfile = project.file("../ios-app/Podfile")
+        }
+    }
+    ```
+    
+    </div>
 
+4. Add the name and path of the Kotlin Pod you want to include in the Xcode project to `Podfile`.
+
+    <div class="sample" markdown="1" theme="idea" mode="ruby" data-highlight-only>
+    
     ```ruby
-    target 'my-ios-app' do
-        pod 'my_kotlin_library', :path => 'path/to/my-kotlin-library'
+    use_frameworks!
+    
+    platform :ios, '9.0'
+    
+    target 'ios-app' do
+            pod 'kotlin_library', :path => '../kotlin-library'
     end
     ```
+    
+    </div>
+
+5. Re-import the project.
+
+### Add a dependency between a Kotlin Pod with an Xcode project with several targets
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`) of
+ your Kotlin project.  
+    This step helps synchronize your Xcode project with Kotlin Pod dependencies by calling `pod install` for your `Podfile`.
+3. Add dependencies to Pod libraries that you want to use in your project with `pod()`.
+4. For each target, specify the minimum target version for the Pod library.
 
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+        tvos()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            ios.deploymentTarget = "13.5"
+            tvos.deploymentTarget = "13.4"
+       
+            pod("AFNetworking", "~> 4.0.0")
+            podfile = project.file("../severalTargetsXcodeProject/Podfile") // specify the path to Podfile
+        }
+    }
+    ```
+    
     </div>
 
-4. Run `pod install` for the Xcode project.
+5. Add the name and path of the Kotlin Pod you want to include in the Xcode project to `Podfile`.
```

Comment:
5. Add the name and path of the Kotlin Pod you want to include in the Xcode project to the `Podfile`.

Reply:
✅